pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = 'bx_py_utils'
-version = '0.0.1.dev0'
+version = "0.0.1.dev0"
```

Comment:
why double quotes?

Reply:
Quotation marks

Reply:
poetry version replaces the Quotation marks :(

Reply:
reported here: https://github.com/python-poetry/poetry/issues/3226